code-challenges.test.js

```diff
@@ -10,47 +10,150 @@
-// Expected output: [-823, 7, 23]
+  })
 
+})
+
+
+// FAIL  ./code-challenges.test.js
+// number
+//   ✕ Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence. (1 ms)
+// oddNum
+//   ✕ takes in an array and returns a new array of only odd numbers sorted from least to greatest (3 ms)77
 
 // b) Create the function that makes the test pass.
 
+const oddNum = (array) => {
+  let newArr =[]
+  for(i = 0; i < array.length; i++){
+      if(array[i] % 2 !== 0 && typeof array[i] === 'number')
+      newArr.push(array[i])
+  }
+  return newArr.sort((a,b) => a-b)
+}
+
+// PASS  ./code-challenges.test.js
+// number
+//   ✓ Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence. (3 ms)
+// oddNum
+//   ✓ takes in an array and returns a new array of only odd numbers sorted from least to greatest (1 ms)
+
+
+// ****** still working on it. 
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
+// describe("hello", () => {
+//     // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+//     it("returns a string that says hi", () => {
+//       //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+//       expect(hello()).toEqual("hi");
+//     });
+//   });
+
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersToAdd1 = [2, 4, 45, 9]
-// Expected output: [2, 6, 51, 60]
 
+// describe("sumArray", () => {
+//   it("takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.", () => {
+const numbersToAdd1 = [2, 4, 45, 9]
+//  Expected output: [2, 6, 51, 60]
 const numbersToAdd2 = [0, 7, -8, 12]
-// Expected output: [0, 7, -1, 11]
-
+// // Expected output: [0, 7, -1, 11]
 const numbersToAdd3 = []
-// Expected output: []
+// // Expected output: []
 
+// expect(sumArray(numbersToAdd1)).toEqual([2, 6, 51, 60])
+// expect(sumArray(numbersToAdd2)).toEqual([0, 7, -1, 11])
+// expect(sumArray(numbersToAdd3)).toEqual([])
+
+//   })
+// }) 
+
+// FAIL  ./code-challenges.test.js
+// number
+//   ✕ Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence. (1 ms)
+// oddNum
+//   ✕ takes in an array and returns a new array of only odd numbers sorted from least to greatest
+// sumArray
+//   ✕ takes in an array and returns an array of the accumulating sum. An empty array should return an empty array. (1 ms)
 
 // b) Create the function that makes the test pass.
+
+const sumArray = (array) => {
```

Comment:
Let's think about what you want to do with the array. What information do you need to get from the input to the expected output?